annotator/annotation.py

```diff
@@ -96,8 +96,28 @@ def _build_query(cls, query=None, offset=None, limit=None, sort=None, order=None
         if query is None:
             query = {}
 
+        # Pop 'before' and 'after' parameters out of the query
+        after = query.pop('after', None)
+        before = query.pop('before', None)
+
         q = super(Annotation, cls)._build_query(query, offset, limit, sort, order)
 
+        # Create range query from before and/or after
+        if before or after:
+            clauses = q['query']['bool']['must']
+
+            # Remove match_all conjunction, because
+            # a range clause is added
+            if clauses[0] == {'match_all': {}}:
```

Comment:
Coming from [here](https://github.com/openannotation/annotator-store/blob/master/annotator/elasticsearch.py#L227)

So, if the query only has before/after parameters. Then the [super call](https://github.com/openannotation/annotator-store/blob/introduce-from-and-to-search-params/annotator/annotation.py#L103) would return a query like this:

```
...
  'must': [
     { 'match_all': {}}
  ]
....
  }
```

And I remove this item, because instead of it we'll have a range query